app/scripts/controllers/incoming-transactions.js

```diff
@@ -267,21 +269,33 @@ export default class IncomingTransactionsController {
       etherscanTransaction.isError === '0'
         ? TRANSACTION_STATUSES.CONFIRMED
         : TRANSACTION_STATUSES.FAILED;
+    const txParams = {
+      from: etherscanTransaction.from,
+      gas: bnToHex(new BN(etherscanTransaction.gas)),
+      nonce: bnToHex(new BN(etherscanTransaction.nonce)),
+      to: etherscanTransaction.to,
+      value: bnToHex(new BN(etherscanTransaction.value)),
+    };
+
+    if (etherscanTransaction.gasPrice) {
+      txParams.gasPrice = bnToHex(new BN(etherscanTransaction.gasPrice));
+    } else if (etherscanTransaction.maxFeePerGas) {
+      txParams.maxFeePerGas = bnToHex(
+        new BN(etherscanTransaction.maxFeePerGas),
```

Comment:
Should we specify the base in these `BN` calls?

... Although, that wasn't done before, so I guess this is fine.

Reply:
I'm curious about this with the comment above regarding wei vs gwei.